### PR TITLE
CON-1062: Update references to the new conclave-core-sdk repo

### PR DESCRIPTION
**Pull request by shamsasari**
Also changed "Conclave SDK" references to either be more specific by adding "Core" or modified in other ways.

The root README has also been cleaned up.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,10 +37,11 @@ Before you submit your Pull Request (PR) consider the following guidelines:
 1. Search [GitHub][github] for an open or closed PR that relates to your submission. You don't want to 
 duplicate existing efforts.
 2. Be sure that an issue describes the problem you're fixing or documents the design for the feature you'd like to add.
+<!--- TODO: Update when the CLA process is finialised. --->
```
Comment (JohnJoser3) [inline CONTRIBUTING.md:40]: finalized

File: conclave-client/src/main/kotlin/com/r3/conclave/client/EnclaveTransport.kt [hunk 2]
```diff
@@ -13,8 +13,8 @@ import java.io.IOException
  *
  * Details such as how to start and shutdown the transport are implementation specific.
  *
- * The Conclave SDK provides [com.r3.conclave.client.web.WebEnclaveTransport] as an implementation for connecting the
- * client to a host which is using the `conclave-web-host`.
+ * [com.r3.conclave.client.web.WebEnclaveTransport] is a HTTP RESTful implementation for connecting the client to a
```
Comment (JohnJoser3) [inline conclave-client/src/main/kotlin/com/r3/conclave/client/EnclaveTransport.kt:16]: án' HTTP (as the pronunciation of HTTP starts with a vowel sound).

File: conclave-web-host/README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 # conclave-web-host
-This module is the web-host section of the Conclave API.
+This is the web-based Conclave host
```
Comment (JohnJoser3) [inline conclave-web-host/README.md:2]: Missing full stop.

File: cpp/linux-sgx/README.md [hunk 4]
```diff
@@ -1,2 +1,5 @@
 # cpp/linux-sgx
-This contains the [Intel SGX SDK](https://github.com/intel/linux-sgx) with additional patches to make it work in Conclave.
+This contains patches to the [Intel SGX SDK](https://github.com/intel/linux-sgx) to make it work with Conclave.
+
+The build compiles the SDK, installs it locally on a build directory so the PSW compilation (`cpp/psw`) can find its
```
Comment (JohnJoser3) [inline cpp/linux-sgx/README.md:4]: The build compiles the SDK and installs it...

File: README.md [hunk 5]
```diff
@@ -1,62 +1,48 @@
-[![Conclave SDK](.github/assets/logo.png)][conclave_website]
+[![Conclave Core SDK](.github/assets/logo.png)][conclave_website]
 
-## Overview
-[Conclave SDK][docs] is an open source platform that makes working with SGX enclaves easy.
+The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves very easily. You can
```
Comment (JohnJoser3) [inline README.md:3]: It's better to remove the 'very'.

The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves easily.

File: README.md [hunk 6]
```diff
@@ -1,62 +1,48 @@
-[![Conclave SDK](.github/assets/logo.png)][conclave_website]
+[![Conclave Core SDK](.github/assets/logo.png)][conclave_website]
 
-## Overview
-[Conclave SDK][docs] is an open source platform that makes working with SGX enclaves easy.
+The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves very easily. You can 
+write your enclave code in high-level languages such as Java, Kotlin and JavaScript.
```
Comment (JohnJoser3) [inline README.md:4]: Please add the Oxford comma.
"write your enclave code in high-level languages such as Java, Kotlin, and JavaScript."

File: README.md [hunk 7]
```diff
@@ -1,62 +1,48 @@
...
-Sources of the documentation are under the directory [docs/](docs/docs).
+## How to use Conclave
+If you want to learn how to use Conclave and build enclaves, take a look at the
+[hello world example and tutorial][hello_world]. You can also refer to the [Conclave documentation][docs] and
+[API docs](https://docs.conclave.net/api/index.html).
 
-### Contributing
-We encourage you to contribute to Conclave. As a contributor please [follow these guidelines](#contributing.md).
+## Contributing
+We encourage you to contribute to Conclave. Have a read of our contribution [guidelines](CONTRIBUTING.md).
```
Comment (JohnJoser3) [inline README.md:18]: Please rewrite 'Have a read of our contribution' as:

Please read our contribution...

File: docs/docs/api-changes.md [hunk 8]
```diff
@@ -43,7 +43,7 @@ they are _not_ thread-safe.
 
 Conclave Init now requires Java 17 to run, and generated projects will target Java 17 by default.
 
-Conclave SDK version is now set at the individual project level, rather at the OS user
+The SDK version is now set at the individual project level, rather at the OS user
```
Comment (JohnJoser3) [inline docs/docs/api-changes.md:46]: The SDK version is now set at the individual project level rather than at the OS user level.

**Review by JohnJoser3 — COMMENTED** [review]
Added some suggestions.

**Review by JohnJoser3 — CHANGES_REQUESTED** [review]
Added my review comments already.

**Comment by shamsasari** [comment]
Comments address, PTAL

**Review by JohnJoser3 — APPROVED** [review]

**Pull request merged**